components/TableColumns/TableColumns.tsx

```diff
@@ -90,6 +105,287 @@ export const columns: ColumnDef<Header>[] = [
      */
     cell: ({ row }) => <div>{row.getValue('text3')}</div>,
   },
+  {
+    accessorKey: 'text4',
+
+    /**
+     * Value of row.
+     * @param {object} column - The column data.
+     * @param {object} column.column - The column definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    header: ({ column }): JSX.Element => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          Surviving?
```

Comment:
nit: This text could be more detailed.

Reply:
It is the header for the table column. Under it, it'll say yes or no in each cell.